inc/appliance_item.class.php

```diff
@@ -200,8 +200,8 @@ static function showItems(Appliance $appliance) {
             }
             echo "<td>" . $item->getTypeName(1) . "</td>";
             echo "<td>" . $item->getLink() . "</td>";
-            echo "<td>" . $item->fields['serial'] ?? "" . "</td>";
-            echo "<td>" . $item->fields['otherserial'] ?? "" . "</td>";
+            echo "<td>" . isset($item->fields['serial']) ?? "" . "</td>";
```

Comment:
Well, I do not understand, the null coalescing operator do exactly the same:
https://www.php.net/manual/en/migration70.new-features.php#migration70.new-features.null-coalesce-op

(french: https://www.php.net/manual/fr/migration70.new-features.php#migration70.new-features.null-coalesce-op)

Reply:
Ok, This happens on string concatenation, the whole expression must be surrounded by parenthesis